### ISSUE TITLE
Cyclists are appearing on "bus only" lanes

### ISSUE THREAD

**Issue by matkoniecz** [dabreegster/abstreet#184]
Is it viable to configure it per city or per lane? For example in Kraków cyclists are banned from bus lanes.

Maybe it would be useful to explicitly tag in OSM via `bicycle:lanes`.

**Comment by dabreegster** [comment]
This happens at https://github.com/dabreegster/abstreet/blob/c20e3701391eff7b679e6919d595c4e6b7ecee38/map_model/src/pathfind/mod.rs#L447. I think it's time to pull `driving_side` from `Map` into some per-city config that also has `bikes_can_use_bus_lanes`. I can set this up today. Detecting it from OSM tags would be useful; I'd have to talk to the community here about doing a partly automated edit.

**Comment by matkoniecz** [comment]
Is it viable to set it per lane? At least here in Kraków it would be one of interesting things to check for impact (what can appear on psv lanes is set by city, on per lane basic - and allowing bicycles on some of them requires "only" adding them to list of vehicle types listed on a traffic sign).

But per city settings also would be very helpful, as current state is mismatching reality.

**Comment by dabreegster** [comment]
> Is it viable to set it per lane?

Definitely, but needs some UI adjustments to show on-map (probably a "bus only" "bikes ok" symbol) and some work in the lane editor. There are more advanced per-lane restrictions that should eventually be supported, like no parking 9-6pm or bus-only from 7-7.

![Screenshot from 2020-07-11 09-57-24](https://user-images.githubusercontent.com/1664407/87229459-f307be80-c35c-11ea-9909-2c7a938b99f5.png)

I'll put together the per-city setting today, and leave the more granular thing as future work.
